frontend/packages/console-shared/src/components/formik-fields/TextColumnField.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { FieldArray } from 'formik';
+import { Flex, FlexItem, FlexModifiers, FormGroup, TextInputTypes } from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+import { global_disabled_color_200 as disabledColor } from '@patternfly/react-tokens';
+import { InputField, useFormikValidationFix } from '@console/shared';
+import MultiColumnFieldFooter from './multi-column-field/MultiColumnFieldFooter';
+import { getFieldId } from './field-utils';
+import { FieldProps } from './field-types';
+
+export interface TextColumnFieldProps extends FieldProps {
+  required?: boolean;
+  isValid?: boolean;
+  name: string;
+  values: string[];
```

Comment:
Why do you need to take values as a prop here? A field component is supposed to connect to the formik context directly using just the name of the field and `useField` hook.